code/datums/gamemode/factions/faction.dm

```diff
@@ -404,7 +404,12 @@ var/list/factions_with_hud_icons = list()
 	roletype = /datum/role/emergency_responder
 	logo_state = "ert-logo"
 	hud_icons = list("ert-logo")
-
+	
+/datum/faction/strike_team/ert/forgeObjectives(var/mission)
```

Comment:
Instead of pasting this, why are you not doing this at the strike_team level?

Reply:
Probably because I was afraid it'd override other strike teams, because I thought nuke ops were a strike team. I'll fix that.